src/v2/Apps/Order/Routes/__tests__/Review.jest.tsx

```diff
@@ -69,14 +72,19 @@ class ReviewTestPage extends OrderAppTestPage {
 
 describe("Review", () => {
   beforeAll(() => {
-    window.sd = { STRIPE_PUBLISHABLE_KEY: "" } as GlobalData
+    window.sd = { STRIPE_PUBLISHABLE_KEY: "" }
     ;(useTracking as jest.Mock).mockImplementation(() => ({
       trackEvent: jest.fn(),
     }))
+    ;(useSystemContext as jest.Mock).mockImplementation(() => ({
```

Comment:
Rather than having to parenthetically typecast each spot where this is used would suggest creating a reusable variable 

```tsx
const mockUseSystemContext = useSystemContext as jest.Mock
```